tests/test_std.rs

```diff
@@ -585,12 +585,6 @@ fn combinations() {
     it::assert_equal((0..1).combinations(1), vec![vec![0]]);
     it::assert_equal((0..2).combinations(1), vec![vec![0], vec![1]]);
     it::assert_equal((0..2).combinations(2), vec![vec![0, 1]]);
-
-    // Infinite iterator should work, even though it doesn't really make sense
-    let mut infinite_combinations = (0..).combinations(2);
-    assert_eq!(infinite_combinations.next(), Some(vec![0, 1]));
-    assert_eq!(infinite_combinations.next(), Some(vec![0, 2]));
-    assert_eq!(infinite_combinations.next(), Some(vec![0, 3]));
```

Comment:
Is there a reason you deleted these tests? They seem valid.

Reply:
Those undos slipped through, let me try again...

Reply:
Actually this was intentional - I replace these tests with some in `adaptors_no_collect.rs`.